src/user/test/user.service.spec.ts

```diff
@@ -38,7 +38,9 @@ describe('UserService', () => {
       expect(createdUser).toEqual({
         id: expect.any(Number),
         email: request.email,
+        enabled2FA: false,
         role: Role.USER,
+        password: undefined,
```

Comment:
I think you don't have to add a `password: undefined`. You could consider `. toDeepEqual` method.